subsys/bluetooth/controller/ll_sw/ull_sync_iso_internal.h

```diff
@@ -6,10 +6,9 @@
 
 int ull_sync_iso_init(void);
 int ull_sync_iso_reset(void);
-struct ll_sync_iso_set *ull_sync_iso_get(uint8_t handle);
-uint8_t ull_sync_iso_handle_get(struct ll_sync_iso_set *sync_iso);
-uint8_t ull_sync_iso_lll_handle_get(struct lll_sync_iso *lll);
-void ull_sync_iso_release(struct ll_sync_iso_set *sync_iso);
+struct ll_sync_iso_set *ull_sync_iso_by_stream_get(uint16_t handle);
+struct lll_sync_iso_stream *ull_sync_iso_stream_get(uint8_t handle);
```

Comment:
Something doesn't sound quite right here. 

`ull_sync_iso_by_stream_get` but then uses a 16-bit handle (I assume connection?)

`ull_sync_iso_stream_get` also uses the word handle, but it's a 8-bit value. Is this the BIG handle? Advertising handle? BIS Index?

Reply:
Ah, now it makes more sense (since you updated `ull_sync_iso_stream_get` to also be `uint16_t`, and I see that you use the same handle for both function.